runtime/mount/endpoints/run.py

```diff
@@ -70,6 +89,23 @@ async def run(ctx: Context) -> HandlerResult:
 
     notebook_id = auth_msg.notebook_id
 
+    s.set_attribute("notebook_id", notebook_id)
+    s.set_attribute("pod_id", pod_id)
+    s.set_attribute("pod_session_id", pod_session_id)
+    s.set_attribute("sess_hash", sess_hash)
+
+    auth_header_regex_match = auth_header_regex.match(auth_msg.token)
+    if auth_header_regex_match is not None:
+        oauth_token = auth_header_regex_match.group("oauth_token")
+        session_token = auth_header_regex_match.group("session_token")
+
+        if oauth_token is not None:
+            auth_data = jwt.decode(oauth_token, options={"verify_signature": False})
```

Comment:
should really expose a helper from `latch-asgi` that allows not checking the signature but reusing the rest of the machinery